spec/controllers/admin/membership_submissions_controller_spec.rb

```diff
@@ -9,7 +9,89 @@
     end
   end
 
-  pending "GET #pending"
-  pending "POST #pending"
-  pending "POST #reject"
+  describe "GET #pending" do
+    describe "when pending submissions are zero" do
+      before do
+        allow(@controller).to receive(:require_admin!).and_return(true)
+        get :pending
+      end
+
+      it "redirects to admin_path" do
+        expect(response).to redirect_to(admin_path)
+      end
+
+      it "sets flash/info" do
+        expect(flash[:info]).to match(/submissions/i)
```

Comment:
Would prefer we added a bit more to the expected match. "submissions" could match a lot of things such as "There are 5 submissions" which is the exact opposite of what we want to ensure here.  I would also change the `it` string to say something about "it sets the flash to indicate there are no submissions"